### ISSUE TITLE
Error in analyser merge_traffic_signs  in a Docker environment

### ISSUE THREAD

**Issue by allier-osm** [osm-fr/osmose-backend#390]
Steps to reproduce:

1. Start a Docker environment with Docker Compose. Options:
```
 --country=comoros --analyser=merge_traffic_signs
```
2. PostgreSQL versions appear in the log during startup:
```
frontend_1_1 |  * Starting PostgreSQL 9.5 database server
backend_1_1 |  * Starting PostgreSQL 10 database server
```
3. An error message appears during analysis:
```
backend_1_1 | 2018-11-10 18:48:47   Batch 8/8.0: regulatory--maximum-speed-limit-60--g3,regulatory--maximum-speed-limit-65--g3,regulatory--maximum-speed-limit-70--g3,regulatory--maximum-speed-limit-75--g3-country=comoros --analyser=merge_traffic_signs
backend_1_1 | 2018-11-10 18:48:47   Page 1
backend_1_1 | 2018-11-10 18:48:47   3 features fetched
backend_1_1 | 2018-11-10 18:48:47   3 keeped
backend_1_1 | 2018-11-10 18:48:47   Analyser_Merge.py:846 sql
backend_1_1 | 2018-11-10 18:48:47   Analyser_Merge.py:608 sql
backend_1_1 | 2018-11-10 18:48:47   Load source into database
backend_1_1 | 2018-11-10 18:48:47   Analyser_Merge.py:616 sql
backend_1_1 | 2018-11-10 18:48:47   Analyser_Merge.py:624 sql
backend_1_1 | 2018-11-10 18:48:47   Analyser_Merge.py:626 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:628 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:629 sql
backend_1_1 | 2018-11-10 18:48:48   Convert data to tags
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:648 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:649 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:695 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:700 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:702 sql
backend_1_1 | 2018-11-10 18:48:48   Retrive OSM item
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:866 sql
backend_1_1 | 2018-11-10 18:48:48   Analyser_Merge.py:888 sql
backend_1_1 | 2018-11-10 18:48:48   sql=CREATE TABLE osm_item AS (
backend_1_1 |                     SELECT
backend_1_1 |                         'W'::char(1) AS type,
backend_1_1 |                         id,
backend_1_1 |                         CASE
backend_1_1 |                             WHEN (tags->'NULL') IS NULL THEN NULL
backend_1_1 |                             ELSE trim(both from regexp_split_to_table(tags->'NULL', ';'))
backend_1_1 |                         END AS ref,
backend_1_1 |                         way_locate(linestring)::geography AS geom,
backend_1_1 |                         ST_Envelope(linestring)::geography AS shape,
backend_1_1 |                         tags
backend_1_1 |                     FROM
backend_1_1 |                         ways
backend_1_1 |                     WHERE
backend_1_1 |                         linestring IS NOT NULL AND
backend_1_1 |                         ST_SetSRID(ST_GeomFromText('POLYGON((3.25345568091956 45.7857577773,3.25345568091956 46.5666096115859,3.32299927093135 46.5666096115859,3.32299927093135 45.7857577773,3.25345568091956 45.7857577773))'), 4326) && linestring AND
backend_1_1 |                         tags != ''::hstore AND
backend_1_1 |                         (tags?'maxspeed' AND tags->'maxspeed' = '5' AND tags?'highway'))
backend_1_1 | 2018-11-10 18:48:48   error on analyse...
backend_1_1 | 2018-11-10 18:48:48     Traceback (most recent call last):
backend_1_1 | 2018-11-10 18:48:48       File "./osmose_run.py", line 251, in run
backend_1_1 | 2018-11-10 18:48:48         analyser_obj.analyser()
backend_1_1 | 2018-11-10 18:48:48       File "/opt/osmose-backend/analysers/Analyser_Merge_Dynamic.py", line 43, in analyser
backend_1_1 | 2018-11-10 18:48:48         analyser_obj.analyser()
backend_1_1 | 2018-11-10 18:48:48       File "/opt/osmose-backend/analysers/Analyser_Osmosis.py", line 180, in analyser
backend_1_1 | 2018-11-10 18:48:48         self.analyser_osmosis_common()
backend_1_1 | 2018-11-10 18:48:48       File "/opt/osmose-backend/analysers/Analyser_Merge.py", line 888, in analyser_osmosis_common
backend_1_1 | 2018-11-10 18:48:48         self.mapping.select.types
backend_1_1 | 2018-11-10 18:48:48       File "/opt/osmose-backend/analysers/Analyser_Osmosis.py", line 436, in run
backend_1_1 | 2018-11-10 18:48:48         self.run0(sql)
backend_1_1 | 2018-11-10 18:48:48       File "/opt/osmose-backend/analysers/Analyser_Osmosis.py", line 390, in run0
backend_1_1 | 2018-11-10 18:48:48         self.giscurs.execute(sql)
backend_1_1 | 2018-11-10 18:48:48       File "/usr/local/lib/python2.7/dist-packages/psycopg2/extras.py", line 141, in execute
backend_1_1 | 2018-11-10 18:48:48         return super(DictCursor, self).execute(query, vars)
backend_1_1 | 2018-11-10 18:48:48     NotSupportedError: set-returning functions are not allowed in CASE
backend_1_1 | 2018-11-10 18:48:48     LINE 7:                             ELSE trim(both from regexp_split...
backend_1_1 | 2018-11-10 18:48:48                                                             ^
backend_1_1 | 2018-11-10 18:48:48     HINT:  You might be able to move the set-returning function into a LATERAL FROM item.
```

**Comment by frodrigo** [comment]
It is now merged.
Osmose is now compatible with Postgres >= 9.3.
There is a know fix to support Postgres < 10.0 : apply the reverse of this diff : https://github.com/osm-fr/osmose-backend/commit/ac247623ebcb5ed54d334df5aab8ac13ab2841e7

**Comment by frodrigo** [comment]
@cc stephankn
